src/content/docs/apm/apm-ui-pages/errors-inbox/errors-inbox.mdx

```diff
@@ -68,6 +68,8 @@ Clicking on a specific error group takes you to the error details page, where yo
 
 The detailed view also allows you to view specific errors. In the top right, you can navigate between the first instance of the error, the last, and any instance inbetween.
 
+Click **Open in IDE** to use CodeStream to find the source of the error in your IDE. This requires that your application is associated with a [repository](/docs/codestream/start-here/codestream-new-relic/#repo-url) and a [commit SHA or release tag](/docs/codestream/start-here/codestream-new-relic/#sha).
+
```

Comment:
@adjohn  Seems like we could say a lot more here.  Thoughts?

Reply:
My philosophy	on this is that we should try to say everything about CodeStream in every place that CodeStream touches. We should let the CodeStream-specific docs do that. That being said, I'm happy to add whatever you all want here.

Reply:
I agree with @paperclypse that we probably don't want to have duplicate "what is codestream" all over the place, which becomes challenging to manage as that messaging evolves.  However, we should at the very least link to the Codestream docs landing page that describes what Codestream is when we mention it.

Reply:
As a minor aside. We will also want to have users send a commit sha/release tag even if they aren't using Codestream, so that they can get better visibility into which versions are affected by which errors (they can do this by using the Attribute analysis functionality today).